### PR TITLE
Add font family metadata

### PR DESCRIPTION
**Pull request by benwiggy**
Add alternative fonts (all clones of Century Schoolbook) to the JSON metadata.

### CODE REVIEW

File: leland_metadata.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
         "textFontFamily": [
             "Edwin",
             "Academico",
-		    "Century Schoolbook",
+		"Century Schoolbook",
```
Comment (Jojo-Schmitz) [inline leland_metadata.json:28]: Still whitespace damage. Use spaces, not tabs

**Comment by benwiggy** [comment]
Better?

**Comment by benwiggy** [comment]
Should I just make a request instead, to add two lines...?

**Comment by oktophonie** [comment]
Thanks for the PR!
To avoid wasting too much of Jojo's time being the GitHub Police, I'll add this change myself (along with another pending metadata tweak).

**Pull request closed without merging**